server/hotrod/src/main/java/org/infinispan/server/hotrod/util/BulkUtil.java

```diff
@@ -21,17 +17,18 @@
    public static final int LOCAL_SCOPE = 2;
 
    public static Iterator<byte[]> getAllKeys(Cache<byte[], ?> cache, int scope) {
```

Comment:
There is just one usage of this (now trivial) method. Is it worth keeping this class around?